sn_networking/src/lib.rs

```diff
@@ -487,17 +487,6 @@ impl Network {
         receiver.await?
     }
 
-    /// Dial the given peer at the given address.
-    pub async fn add_to_routing_table(&self, peer_id: PeerId, peer_addr: Multiaddr) -> Result<()> {
-        let (sender, receiver) = oneshot::channel();
-        self.send_swarm_cmd(SwarmCmd::AddToRoutingTable {
-            peer_id,
-            peer_addr,
-            sender,
-        })?;
-        receiver.await?
-    }
-
```

Comment:
If we are not using the function, then I think we can get rid of the corresponding `SwarmCmd` as well?